receipt_dynamo/receipt_dynamo/entities/receipt_place.py

```
@@ -0,0 +1,441 @@
+    # Convert receipt_id: boto3 deserializes numbers as Decimal, need int
+    if "receipt_id" in filtered_item:
+        if isinstance(filtered_item["receipt_id"], Decimal):
+            filtered_item["receipt_id"] = int(filtered_item["receipt_id"])
+        elif isinstance(filtered_item["receipt_id"], str):
+            try:
+                filtered_item["receipt_id"] = int(filtered_item["receipt_id"])
+            except (ValueError, TypeError):
+                pass
+
+    # Handle None values for string fields that should be empty strings
+    for field in ["geohash", "merchant_category", "formatted_address", "short_address", "phone_number", "phone_intl", "website", "maps_url", "business_status", "plus_code", "validated_by", "reasoning", "places_api_version"]:
```

Ruff (0.14.8)

Import `field` from line 24 shadowed by loop variable

(F402)